### PR TITLE
[nt] ace code editor polish

### PR DESCRIPTION
**Pull request by nathaniel-mage**
# Summary

### Improvements to the code editor code
- Refactor the repeated code into a const file
- Flesh out more of the parameters
# Tests

### Export pages (no highlighting lines/gutter)
image

### Suggestions List
image
image

### Custom Code Actions
image

### Known bugs
- When going from custom to filter without closing the form it doesn't change the values
### Notes on implementation
- Due to the nature of NextJS and the dynamic call, are we able to change the Code Editor into a component instead?
- It feels really annoying to have to create a new dynamic function, import the same builds in order to get it 
- Also, I was unable to get the cursor to completely hide, but I made it slimmer so hopefully it's less noticeable. It still will highlight/box the current text it's on though, like a real code editor.

cc: @johnson-mage

### CODE REVIEW

**Comment by nathaniel-mage** [comment]
The bug is on master not just this branch.

**Comment by johnson-mage** [comment]
> are we able to change the Code Editor into a component instead?

why don't you try this?

**Comment by johnson-mage** [comment]
> When going from custom to filter without closing the form it doesn't change the values

This is a pretty important bug to fix. did you add a story?

**Comment by johnson-mage** [comment]
The cursor doesn't look any less visible after making it slimmer, but ideally, we still wanna hide it initially. Add a story for that if you havent already.

**Review by johnson-mage — APPROVED** [review]

File: mage_ai/frontend/components/datasets/Export/ace.style.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import styled, { css } from 'styled-components';
+
+import light from '@oracle/styles/themes/light';
+import { BORDER_RADIUS, BORDER_RADIUS_SMALL } from '@oracle/styles/units/borders';
+import { UNIT } from '@oracle/styles/units/spacing';
+
+
+export const AceStyle = css<any>`
```
Comment (johnson-mage) [inline mage_ai/frontend/components/datasets/Export/ace.style.tsx:8]: Is this used anywhere? If not, why do we have it?

Comment (johnson-mage) [reply mage_ai/frontend/components/datasets/Export/ace.style.tsx:8]: @nathaniel-mage

Comment (nathaniel-mage) [reply mage_ai/frontend/components/datasets/Export/ace.style.tsx:8]: I was experimenting with trying to access its children from the package. It didn't work. Will delete the file.

**Pull request merged**